src/TorchSharp/NN/Losses.cs

```diff
@@ -12,16 +12,56 @@ namespace TorchSharp
 {
     using Modules;
 
-    public class Loss : nn.Module
+    public abstract class Loss<T1, T2, TResult> : nn.Module<T1, T2, TResult>
```

Comment:
Are there any losses, that do not return `Tensor`? I am curious if it makes sense to have it as a generic parameter.

Reply:
You can imagine some that return "double" or something like that, even if there's none that currently return anything but a tensor.